src/h5web/visualizations/nexus/utils.ts

```diff
@@ -19,44 +28,11 @@ import {
   isScaleType,
 } from '../shared/utils';
 
-export function isNxInterpretation(attrValue: HDF5Value): boolean {
-  return (
-    attrValue &&
-    typeof attrValue === 'string' &&
-    NX_INTERPRETATIONS.includes(attrValue)
-  );
-}
-
 export function getAttributeValue(
   entity: HDF5Dataset | HDF5Group,
   attributeName: NxAttribute
 ): HDF5Value | undefined {
-  if (!entity.attributes) {
-    return undefined;
-  }
-
-  return entity.attributes.find((attr) => attr.name === attributeName)?.value;
-}
-
-export function getLinkedEntity(
-  entityName: string,
-  group: HDF5Group,
-  metadata: HDF5Metadata
-): HDF5Entity | undefined {
-  const childLink = group.links?.find((l) => l.title === entityName);
-  return getEntity(childLink, metadata);
-}
-
-export function getNxAxisNames(group: HDF5Group): (string | undefined)[] {
-  const axisList = getAttributeValue(group, 'axes');
-
-  if (!axisList) {
-    return [];
-  }
-
-  const axisNames = typeof axisList === 'string' ? [axisList] : axisList;
-  assertArray<string>(axisNames);
-  return axisNames.map((a) => (a !== '.' ? a : undefined));
+  return entity.attributes?.find((attr) => attr.name === attributeName)?.value;
```

Comment:
Sorry, I ended up using `?.`. 😅  It felt very inconsistent since we use `?.` in the same line already, and we use this syntax in other similar utilities.

Reply:
No worries, it looks good